server/src/main/java/io/unitycatalog/server/persist/CatalogRepository.java

```diff
@@ -55,6 +55,8 @@ public CatalogInfo addCatalog(CreateCatalog createCatalog) {
         CatalogInfoDAO catalogInfoDAO = CatalogInfoDAO.from(catalogInfo);
         PropertyDAO.from(catalogInfo.getProperties(), catalogInfoDAO.getId(), Constants.CATALOG)
             .forEach(session::persist);
+        RepositoryUtils.attachProperties(
+            catalogInfo, catalogInfo.getId(), Constants.CATALOG, session);
```

Comment:
These would already be present from [here](https://github.com/unitycatalog/unitycatalog/pull/190/files#diff-7c78786fc147d2887740af532f7538cd50e04960aa35c7a77e1c90cbb210c096R46), we do not need this change